### ISSUE TITLE
Setting bias to zero in gblinear

### ISSUE THREAD

**Issue by jaguerrerod** [dmlc/xgboost#8848]
How can I set the bias to 0 in gblinear booster now?
And what happened with lambda_bias? (a way to force bias to zero)
Documentation about gblinear is scarce or I don't know where looking for.

**Comment by trivialfis** [comment]
It's not supported.

**Comment by jaguerrerod** [comment]
So there isn't a way to force a linear model without offset?

**Comment by trivialfis** [comment]
No, we spend most of the time on tree models and the linear model is less attended.

**Comment by jaguerrerod** [comment]
I set base_score to 0 and getting very small values of offset. My dataset is centered at 0 (mean response is zero).
Could this works in general? How is computed the offset in gblinear?

**Comment by trivialfis** [comment]
Closing as GBLinear has been deprecated https://github.com/dmlc/xgboost/issues/12023.
